cosmoz-moneyhelper-behavior.html

```diff
@@ -112,6 +112,30 @@
 	 * @demo
 	 */
 	Cosmoz.MoneyHelperBehavior = {
+		/**
+		 * Compare amounts.
+		 * @param {object} amount1 Amount 1.
+		 * @param {object} amount2 Amount 2.
+		 * @param {number} precision Decimal precision, defaults to 2 decimals.
+		 * @returns {boolean} Comparison result.
```

Comment:
Be more expressive.. what's the comparison result? If they equal eachother? Then write that rather.

Reply:
Added a more descriptive text.